src/services/dashboards/dashboard-create/DashboardCreatePage.vue

```diff
@@ -88,8 +123,22 @@ export default {
         const state = reactive({
             dashboardScope: DASHBOARD_SCOPE.DOMAIN as DashboardScope,
             dashboardViewerType: DASHBOARD_VIEWER.PUBLIC as DashboardViewer,
+            steps: [
+                { step: 1, name: 'choose scope and viewer', description: 'Choose dashboard scope and viewer.' },
+                { step: 2, name: 'choose template', description: 'Choose a template or an existing dashboard to get started quickly.' },
+            ],
+            currentStep: 1,
+            valid: computed(() => {
```

Comment:
nitpick: I think it would be better to write "isValid" according to the code convention!